src/server/svr_func.c

```diff
@@ -656,11 +656,16 @@ cred_name_okay(attribute *pattr, void *pobj, int actmode)
 int
 poke_scheduler(attribute *pattr, void *pobj, int actmode)
 {
-	if (actmode == ATR_ACTION_ALTER) {
-		if (pattr->at_val.at_long)
-			set_scheduler_flag(SCH_SCHEDULE_CMD);
+	if ( pobj == &server || pobj == dflt_scheduler) {
```

Comment:
We will consider your input. But for the time being since this function is common action function for both server and scheduler and hence keeping it here. Also EDD update is in progress for updating/validating default scheduler attributes.